keycloak/keycloak-admin-ui
# Realm settings(keys): add ECDSA provider details

## Motivation

Closes #849 ; Towards #639

Also https://issues.redhat.com/browse/APPDUX-1022


## Brief Description

Adds details/edit functionality for ecdsa-generated providers in keys tab of realm settings.

## Verification Steps


1. Go to Realm settings -> Keys -> Providers
2. Select "ecdsa-generated", populate fields in modal, and save
3. Find the new provider you just created in the list and click on it.
4. See the new "ecdsa-generated" details page. 
5. Edit some fields, save, and navigate elsewhere in the app.
6. Navigate back to your updated ecdsa-generated provider and verify that the data is saved correctly.

## Checklist:


## Additional Notes
